.github/workflows/github_metrics.yml

```
@@ -194,6 +194,59 @@ jobs:
                 - "project:haystack-core-integrations"
                 - "type:health"
 
+  non-haystack-core-integrations-pypi-metrics:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package:
+          - apify-haystack
+          - openinference-instrumentation-haystack
+          - assemblyai-haystack
+          - context-haystack
+          - couchbase-haystack
+          - deepl-haystack
+          - docling-haystack
+          - duckduckgo-api-haystack
+          - elevenlabs-haystack
+          - lancedb-haystack
+          - marqo-haystack
+          - mastodon-fetcher-haystack
+          - milvus-haystack
+          - mixedbread-ai-haystack
+          - needle-haystack-ai
+          - neo4j-haystack
+          - notion-haystack
+          - takeoff-haystack
+          - vllm-haystack
+          - voyage-embedders-haystack
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Hatch
+        run: pip install hatch
+
+      - name: Get PyPI downloads
+        id: pypi-downloads
+        working-directory: collector
```

Do you think we might need this here to allow other jobs to run if something fails.
```suggestion
        working-directory: collector
        continue-on-error: true

```

---

Right! Thanks for explaining.

---

Thanks for sharing the idea! Let's hope the workflow doesn't fail too often.🤞